_includes/macros.html

```diff
@@ -285,3 +268,70 @@ <h1 class="h5 {{'u-visually-hidden' if hide_heading}}">Share this page</h1>
 
 {% endmacro %}
 
+
+{# ==========================================================================
+
+   YouTube()
+
+   ==========================================================================
+
+   Description:
+
+   Builds YouTube markup when given:
+
+   options (optional): An object used to customize the YouTube markup.
+
+   options.image_res: The image resolution to use for the YouTube image.
+
+   options.video_id: An id associated with a YouTube video.
+
+   ========================================================================== #}
+
+{% macro YouTube(options) -%}
+    {% set image_res = options.image_res | default('default') %}
+    {% set image_res_options = {
+          'default' : 'default.jpg',
+          'standard': 'sddefault.jpg',
+          'medium'  : 'mqdefault.jpg',
+          'max'     : 'maxresdefault.jpg',
+          'high'    : 'hqdefault.jpg'
+        }
+    %}
+    {% set video_id  = options.video_id %}
+    {% set video_url = 'http://www.youtube.com/embed/' + video_id %}
+    {% set image_url = options.image_url | default(
+                                              'http://img.youtube.com/vi/'
```

Comment:
Maybe make this `https` like https://github.com/cfpb/cfgov-refresh/pull/552/files#diff-ae11b16fd39f830f6f52d38e54892811R329